HoaryFox/Member/FrameBreps.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using STBReader;
+using STBReader.Member;
+using STBReader.Model;
+using STBReader.Section;
+using Rhino.Geometry;
+
+namespace HoaryFox.Member
+{
+    public class FrameBreps
+    {
+        private readonly StbData _stbData;
+
+        public FrameBreps(StbData stbData)
+        {
+            _stbData = stbData;
+        }
+        
+        public List<Brep> Slab(StbSlabs slabs)
+        {
+            var brep = new List<Brep>();
+            var count = 0;
+
+            foreach (List<int> nodeIds in slabs.NodeIdList)
+            {
+                var index = new int[nodeIds.Count];
+                var pts = new Point3d[nodeIds.Count];
+                double offset = slabs.Level[count];
+
+                for (var i = 0; i < nodeIds.Count; i++)
+                {
+                    index[i] = _stbData.Nodes.Id.IndexOf(nodeIds[i]);
+                    pts[i] = new Point3d(_stbData.Nodes.X[index[i]], _stbData.Nodes.Y[index[i]], _stbData.Nodes.Z[index[i]] + offset);
+                }
+                
+                brep.Add(CreateBreps.PlaneWithOpens(_stbData, pts, null));
+                count++;
+            }
+
+            return brep;
+        }
+
+        public List<Brep> Wall(StbWalls walls)
+        {
+            var brep = new List<Brep>();
+            var count = 0;
+
+            foreach (List<int> nodeIds in walls.NodeIdList)
+            {
+                var index = new int[nodeIds.Count];
+                var pts = new Point3d[nodeIds.Count];
+
+                for (var i = 0; i < nodeIds.Count; i++)
+                {
+                    index[i] = _stbData.Nodes.Id.IndexOf(nodeIds[i]);
+                    pts[i] = new Point3d(_stbData.Nodes.X[index[i]], _stbData.Nodes.Y[index[i]], _stbData.Nodes.Z[index[i]]);
+                }
+
+                brep.Add(CreateBreps.PlaneWithOpens(_stbData, pts, walls.Opens[count]));
+                count++;
+            }
+
+            return brep;
+        }
+
+        public List<Brep> Frame(StbFrame frame)
```

Comment:
Codacy found an issue: [Rename the parameter 'frame' so that it does not duplicate the method name.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6397782)